tests/testing_common.py

```diff
@@ -68,7 +68,7 @@
 class ClassInstantier(OrderedDict):
     def __getitem__(self, key, *args, **kwargs):
         # check if any of the kwargs is inside the config class kwargs
-        if any([kwarg in self[key][1] for kwarg in kwargs]):
+        if any(kwarg in self[key][1] for kwarg in kwargs):
```

Comment:
Curious why this is needed?

Reply:
ruff raises quality issue.

Reply:
Ok I see, I think it's ok to merge it as it is,